_checks/styles/style-rules-prod

```diff
@@ -35,4 +35,4 @@ rule 'MD038'
 exclude_rule 'MD039'
 rule 'MD040'
 exclude_rule 'MD041'
-exclude_rule 'MD046'
+rule 'MD046', :style => :fenced
```

Comment:
```suggestion
rule 'MD046', :style => :fenced # code block style
```

Reply:
I shouldn't have committed that change. We don't want to turn the rule on.